dist/services/StructuralMetadataUtils.js

```diff
@@ -156,51 +156,43 @@ function () {
     value: function buildSMUI(allItems, duration) {
       var _this2 = this;
 
-      // Regex to match hh:mm:ss, mm:ss OR seconds(as a float)/minutes(as an int)
-      var regexHHMMSS = /^([0-9]*:[0-9]*:[0-9]*.[0-9]*)$/i;
-      var regexMMSS = /^([0-9]*:[0-9]*)$/i;
-      var regexSS = /^([0-9]*.[0-9]*)$/i; // Convert file duration to seconds
+      // Regex to match hh:mm:ss.ms, mm:ss.ms OR minutes(as an int)
+      var regexHHMMSS = /^([0-9]*:[0-9]*(:|.)[0-9]*.[0-9]*)$/i;
+      var regexMMSS = /^([0-9]*:[0-9]*)$/i; // Convert file duration to seconds
```

Comment:
I think this is still not quite right since from my reading all times require a `:` so the `ss.sss` case isn't being supported.
I think in the end we need these patterns supported:
ss(.sss)
mm:ss(.sss)
hh:mm:ss(.sss)
so this might be more like this ruby regex:
`^(\d{0,2}:)?([012345]?\d:)?([012345]?\d)(\.\d{0,3})?$`
https://rubular.com/r/KJ9J4WNnOcQ6iD